pygmt/figure.py

```diff
@@ -259,14 +259,15 @@ def savefig(
 
         Supported file formats and their extensions:
 
-        - PNG (``.png``)
-        - JPEG (``.jpg`` or ``.jpeg``)
-        - PDF (``.pdf``)
         - BMP (``.bmp``)
-        - TIFF (``.tif``)
-        - GeoTIFF (``.tiff``)
         - EPS (``.eps``)
+        - GeoTIFF (``.tiff``)
+        - JPEG (``.jpg`` or ``.jpeg``)
         - KML (``.kml``)
+        - PDF (``.pdf``)
+        - PNG (``.png``)
+        - PPM (``.ppm``)
```

Comment:
I am not really familiar with the `ppm` format, but maybe we should add the `ppm` format also to the list of raster images for which the `anit_alias` parameter can be used (see line 287):

```
anti_alias: bool
            If ``True``, will use anti-aliasing when creating raster images
            (BMP, PNG, JPEG, TIFF, and GeoTIFF). More specifically, it passes
            the arguments ``"t2"`` and ``"g2"`` to the ``anti_aliasing``
            parameter of :meth:`pygmt.Figure.psconvert`. Ignored if creating
            vector graphics.
```

Reply:
In 0c8184710ea93a59ce6e3b2027114edb53e63421, I've revised the description of supported images formats and clarify the raster and vectors images that PyGMT supports.

Reply:
@yvonnefroehlich OK to approve?

Reply:
Looks good to me!